AST.fs

```diff
@@ -281,45 +314,46 @@ module Pretty =
                printBlock pView pCmd b ]
 
     /// Pretty-prints commands with the given indent level (in spaces).
-    let rec printCommand pView =
+    let rec printCommandType pView =
         function
         (* The trick here is to make Prim [] appear as ;, but
            Prim [x; y; z] appear as x; y; z;, and to do the same with
            atomic lists. *)
-        | Prim { PreAssigns = ps
-                 Atomics = ts
+        | CommandTypes.Prim { PreAssigns = ps;
```

Comment:
Hmm, the semicolons here should be unnecessary and automatically inserted due to the newlines.  Actually, maybe roll this up into one line?